slick-testkit/src/test/scala/slick/test/jdbc/hikaricp/SlickInUseCountTest.scala

```diff
@@ -84,7 +85,11 @@ class SlickInUseCountTest extends AsyncTest[JdbcTestDB] {
     val threadPoolExecutor = threadPoolExecutorField.get(asyncExecutor)
 
     val queue = threadPoolExecutor.getClass.getMethod("getQueue").invoke(threadPoolExecutor)
-    val inUseCountField = queue.getClass.getDeclaredField("slick$util$ManagedArrayBlockingQueue$$inUseCount")
+    val inUseCountField = Seq("inUseCount", "slick$util$ManagedArrayBlockingQueue$$inUseCount").collectFirst{name =>
+	Try(queue.getClass.getDeclaredField(name)) match{
+            case Success(field) => field
```

Comment:
Does this work as intended? Only having a Success clause leades to a MatchError in case the field can't be found.